src/org/isf/accounting/service/AccountingBillIoOperationRepository.java

```diff
@@ -24,6 +26,16 @@
 
 	@Query(value = "SELECT * FROM BILLS WHERE DATE(BLL_DATE) BETWEEN :dateFrom AND :dateTo", nativeQuery = true)
 	List<Bill> findAllWhereDates(@Param("dateFrom") Timestamp dateFrom, @Param("dateTo") Timestamp dateTo);
+	
+	@Query(value = "SELECT * FROM BILLS  WHERE  (DATE(BLL_DATE) BETWEEN :dateFrom AND :dateTo ) "
+			+" AND ( BLL_ID_PAT= :patientCode)", nativeQuery = true)
+	ArrayList<Bill> findByDateAndPatient(@Param("dateFrom")GregorianCalendar dateFrom, @Param("dateTo") GregorianCalendar dateTo, @Param("patientCode")Integer patientCode);
+	
+	@Query(value = "SELECT * FROM BILLS "
+			+"WHERE BLL_STATUS='O' "
+			+"  AND BLL_ID_PAT=:patID ", nativeQuery = true)
+	ArrayList<Bill> findAllPendindBillsByPatient(@Param("patID")int patID);
+
```

Comment:
is this part concerning, and in general all the changes to the billing module, concerning this issue?